.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -122,11 +121,10 @@ steps:
       -e OS_DISK_URI=${OS_DISK_URI} \
       -e MANAGED_SIG_ID=${MANAGED_SIG_ID} \
       -e AZURE_LOCATION=${AZURE_BUILD_LOCATION} \
-      -e CONTAINER_RUNTIME=$(CONTAINER_RUNTIME) \
       -e WINDOWS_SKU=$(WINDOWS_SKU) \
       -e OS_TYPE="Windows" \
       -e MODE=$(MODE) \
-      -e FEATURE_FLAGS=$(FEATURE_FLAGS) \
+      -e FEATURE_FLAGS=${FEATURE_FLAGS} \
```

Comment:
It seems like that there is no `FEATURE_FLAGS` in windows pipelines. Will an empty value be set here?

Reply:
I think so. I refer to the setting of `ARCHITECTURE`, which is also not used in windows pipelines.